scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/XmlParser.scala

```diff
@@ -123,15 +123,13 @@ class XmlParser(Block: P0,
   }
 }
 
-case class RangePosition(from: Int, to: Int)
```

Comment:
I personally dislike `(Int, Int)` tuples, it's not clear what each field does. I would prefer to give it a more precise name like `XmlTokenRange`, or reuse Position.Range. It adds mental cycles to see what `List.newBuilder[(Int, Int)]` means.